### PR TITLE
Replace uses of tostring with tobytes

### PR DESCRIPTION
**Pull request by saimn**
`ndarray.tostring` will be deprecated in Numpy 1.19 : numpy/numpy#15867

### CODE REVIEW

File: astropy/units/tests/test_quantity_array_methods.py [hunk 1]
```diff
@@ -511,6 +511,7 @@ def test_not_implemented(self):
         with pytest.raises(NotImplementedError):
             q1.tolist()
         with pytest.raises(NotImplementedError):
+            # FIXME: tobytes instead ?
             q1.tostring()
```
Comment (saimn) [inline astropy/units/tests/test_quantity_array_methods.py]: @mhvk - What to do about this one? Should `tobytes` be added here and in `astropy/units/quantity.py` ?

**Comment by pllim** [comment]
As long as tests pass, the `votable` change is fine by me. But I don't see the Travis CI job listed here. Where did it go?

File: astropy/units/tests/test_quantity_array_methods.py (hunk 1 above)
Comment (mhvk) [reply astropy/units/tests/test_quantity_array_methods.py]: Yes, I think it was an oversight not to have those. If you could add those, that would be great!

Comment (saimn) [reply astropy/units/tests/test_quantity_array_methods.py]: @mhvk - Ok, done.

**Pull request merged**